### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by TheoBoschetti**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]
ok

File: frontend/src/Components/Game.jsx [hunk 1]
```diff
@@ -157,85 +156,133 @@ class Game extends React.Component {
...
-    } else if (this.state.buttonPressed.axes[1] === "-1.00") {
+    } else if (
+      this.state.buttonPressed.axes[1] === "-1.00" &&
+      this.state.direction === "up"
+    ) {
+      this.setState({ pressKey: this.state.pressKey + 1 });
       newPosition = y - 1;
       newDirection = "up";
       if (this.isMovePossible(x, y - 1)) {
+        this.setState({ pressKey: this.state.pressKey + 1 });
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx]: Why do a separate `setState` for pressKey ? It'd be better to update it in the same as below.
(Same comment for several occurences in this file)

**Review by LoicBrassart — DISMISSED** [review]
Didn't see any major Evil in this

_NB: In any company, I think this PR would be immediately dismissed because of the sheer volume of code and the miscellaneous topics covered in it_

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**